packages/model-typings/src/models/IRoomsModel.ts

```diff
@@ -12,18 +12,7 @@ export interface IRoomsModel extends IBaseModel<IRoom> {
 
 	getMostRecentAverageChatDurationTime(numberMostRecentChats: any, department: any): Promise<any>;
 
-	findByNameOrFnameContainingAndTypes(
-		name: any,
-		types: any,
-		discussion?: boolean,
-		teams?: boolean,
-		showOnlyTeams?: boolean,
-		options?: any,
-	): any;
-
-	findByTypes(types: any, discussion?: boolean, teams?: boolean, onlyTeams?: boolean, options?: any): any;
-
-	findByNameOrFnameContaining(name: any, discussion?: boolean, teams?: boolean, onlyTeams?: boolean, options?: any): any;
+	findByNameOrFnameContainingAndTypes(name: any, types: any, discussion?: boolean, teams?: boolean, options?: any): any;
```

Comment:
Is it possible to correct the typing since you've modified this line?

Reply:
Heyy, I rewrote all this file to typescript on [this PR](https://github.com/RocketChat/Rocket.Chat/pull/28260)